proto/babylon/incentive/events.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+package babylon.incentive;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/babylonlabs-io/babylon/v2/x/incentive/types";
+
+// EventsPowerUpdateAtHeight stores an slice of events that happened at
+// some specific block height, it is indexed by the block height.
+message EventsPowerUpdateAtHeight {
+  repeated EventPowerUpdate events = 1;
+}
+
+// EventPowerUpdate is an event that affects voting power distribution
+// of BTC rewards, it is indexed by the babylon block height which
+// they were processed at the finality voting power table.
+message EventPowerUpdate {
+  // ev is the event that affects voting power distribution
+  oneof ev {
+    // BtcActivated means a BTC delegation becomes active.
+    EventBTCDelegationActivated btc_activated = 1;
```

Comment:
I remember RFC mentioned we will need also slashed event. Ultimately is it unnecessary?

Reply:
I thought it was too much change for now, since the slashing of the rewards is also not wired at v1.1